devicetypes/fibargroup/fibaro-double-switch-2-zw5.src/fibaro-double-switch-2-zw5.groovy

```diff
@@ -329,27 +329,29 @@ def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cm
 def zwaveEvent(physicalgraph.zwave.commands.meterv3.MeterReport cmd, ep=null) {
 	logging("${device.displayName} - MeterReport received, value: ${cmd.scaledMeterValue} scale: ${cmd.scale} ep: $ep","info")
 	log.debug "cmd: "+cmd
+	def value = cmd.scaledMeterValue.doubleValue()
+	def defaultValue = 0.0
 	if (ep==1) {
 		switch (cmd.scale) {
 			case 0:
-				sendEvent([name: "energy", value: cmd.scaledMeterValue, unit: "kWh"])
+				sendEvent([name: "energy", value: value, unit: "kWh"])
 				break
 			case 2:
-				sendEvent([name: "power", value: cmd.scaledMeterValue, unit: "W"])
+				sendEvent([name: "power", value: value, unit: "W"])
 				break
 		}
-		multiStatusEvent("${(device.currentValue("power") ?: "0.0")} W | ${(device.currentValue("energy") ?: "0.00")} kWh")
+		multiStatusEvent("${(device.currentValue("power")?.doubleValue() ?: defaultValue)} W | ${(device.currentValue("energy")?.doubleValue() ?: defaultValue)} kWh")

```

Comment:
This DTH is missing the definition of `attribute "multiStatus", "string"` so may as well fix that while you're in here.